optimum/intel/openvino/modeling_base.py

```diff
@@ -835,12 +839,6 @@ def _export(
             )
             compile_only = False
 
-        # If load_in_8bit and quantization_config not specified then ov_config is set to None and will be set by default in convert depending on the model size
-        if load_in_8bit is None and not quantization_config:
-            ov_config = None
-        else:
-            ov_config = OVConfig(dtype="fp32")
```

Comment:
No longer needed because there is no quantization performed at main_export anymore. Replaced `dtype="fp32"` with `dtype="auto"` because in most scenarios we already export models with "auto" dtype and there were no issues with it.